webpack.config.js

```diff
@@ -16,18 +15,9 @@ module.exports = {
   output: {
     filename: 'main.js',
     path: path.resolve(__dirname, 'dist'),
-    assetModuleFilename: (pathData) => {
-      const { filename } = pathData;
-
-      if (filename.endsWith('.ts')) {
-        return '[name]-[hash].js'
-      } else {
-        return '[hash][ext][query]';
-      }
-    }
   },
   plugins: [new HtmlWebpackPlugin()],
   resolve: {
-    extensions: ['.tsx', '.ts', '.js'],
+    extensions: ['.tsx', '.ts', '...'],
```

Comment:
Try to not override default `extensions`, just perpend own extensions to default using `...` syntax